### PR TITLE
attrib: add page

### PR DESCRIPTION
**Pull request by yanuehara**
- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/windows/attrib.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Display the permission of the files in current directory and sub-directories:
+
+`attrib /S`
+
+- Display the permission of the files and folders in current directory and sub-directories:
+
+`attrib /S /D`
+
+- Add read-only attribute to file:
```
Comment (sbrl) [inline pages/windows/attrib.md]: Perhaps `Add the read-only` would read better?

File: pages/windows/attrib.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# attrib
+
+> Displays or changes file and directories attributes.
+
+- Display the permission of the files in current directory:
```
Comment (sbrl) [inline pages/windows/attrib.md]: You mention attributes in the description of the command, but you use the word permission here? This sounds a bit confusing.

File: pages/windows/attrib.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Display the permission of the files and folders in current directory and sub-directories:
+
+`attrib /S /D`
+
+- Add read-only attribute to file:
+
+`attrib +R {{document.txt}}`
+
+- Remove system and hidden atrtibute of a file:
```
Comment (sbrl) [inline pages/windows/attrib.md]: Perhaps `Remove the system and hidden attributes of a file` would read better here?

File: pages/windows/attrib.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`attrib -S -H {{document.txt}}`
+
+- Add hidden attribute to a folder:
```
Comment (sbrl) [inline pages/windows/attrib.md]: Same here - perhaps `Add the hidden attribute to` would read better?

File: pages/windows/attrib.md (hunk 2 above)
Comment (agnivade) [reply pages/windows/attrib.md]: I think "attributes" sound better. Because we have system, hidden and all that.

**Comment by agnivade** [comment]
Going to merge this since there has been no explicit disapproval.

**Pull request merged**